backend/entities/__init__.py

```diff
@@ -14,6 +14,7 @@
 
 
 from .entity_base import EntityBase
+from .courses import *
```

Comment:
@KrisJordan thank you for pointing this out! This was actually a change I had to make because for some reason, the entities were not being recognized in the relationship field with `UserEntity`. If we were to not include this relationship at the entity level though (as might be suggested below), then we can go ahead and remove this.

Reply:
Removed this in the latest commit.